src/psyclone/tests/config_test.py

```diff
@@ -33,7 +33,7 @@
 # -----------------------------------------------------------------------------
 # Author: A. R. Porter, STFC Daresbury Lab
```

Comment:
Not yours, but as a final step, please could you rename this file to `configuration_test.py` so that it conforms to our naming scheme.

Reply:
OK, to run this past you - there are unfortunately two options:
1. The class is actually called `Config`, so from a coding style point of view the file should be called `config.py` (not `configuration.py`), and the test is actually named correctly :(
2. Alternatively, we could call the class `Configuration` instead (and rename the test)

Unfortunately, both these correct solutions will make for e huge patch (since all imports or `Config` uses needs to be updated everywhere :( ).

So I assume you prefer just renaming the test? I am happy to do any of the above (I believe editors can rename a class everywhere), but am aware of the potential impact (I could do this after Sergi's patch goes in?). Just let me know what you prefer.

Reply:
Oh dear. Let's just rename the test then :-)